portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,214 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+     <link rel="stylesheet" href="style.css">
+    <title>Edson Casimiro</title>
+    <!--adding google fonts and fontawesome-->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.1/css/all.css">
+</head>
+<body>
+
+    <nav>
+        <div class="logo">
+            Edson Casimiro
+        </div>
+        <div class="hamburger">
+            <div class="line1"></div>
+            <div class="line2"></div>
+            <div class="line3"></div>
+        </div>
+        <ul class="nav-links">
+            <li><a href="#home">Home</a></li>
+            <li><a href="#course">Courses&skills</a></li>
+            <li><a href="#experience">Experience</a></li>
+            <li><a href="#project">Project</a></li>
+
+        </ul>
+    </nav>
+<p id="home"></p>
+
+
+    <div class="start" style="overflow:auto;">
+        <div class="intro1" >
+            <img src="me.jpg" alt="edson casimiro">
+            <h3>Computer Engineering Student</h3>
+            <p><span><i class="material-icons" id="btn">home</i>Ks, Angola</span> <span><i class="material-icons"
+                        id="btn">location_on</i>Istanbul, Turkey</span></p>
+            <p><span><i class="material-icons" id="btn">translate</i></span>Portuguese, English, Turkish </p>
+
+            <br>
+            <h1>
+                <a href="https://www.linkedin.com/in/edson-casimiro-435500137/"
+                    style="textdecoration:none; color:black;">
+                    <i class="fab fa-linkedin"></i>
+                </a>
+                <a href="https://github.com/HiEdson" style="textdecoration:none; color:black;">
+                    <i class="fab fa-github"></i>
+                </a>
+                <a href="mailto: edsoncasimiro264@gmail.com" style="textdecoration:none; color:black;">
+                    <i class="fas fa-envelope"></i>
+                </a>
+
+            </h1>
+
+
+        </div>
+
+
+
+        <div class="intro2" >
+            <h1>Edson Casimiro</h1>
+            <p>I am an undergraduate student in the Department of Computer Engineering at Istanbul University
+                graduating in May 2023.
+                previously, i studied high school in angola with the major in electricity, electronics and
+                telecommunications.</p>
+            <p>I am passionate about artificial intelligence algorithms and analyzing safe ways to apply them to web
+                systems as well
```

Comment:
It's good that you're splitting up long lines with newlines, but some of these appear a bit uneven. Maybe consider making the linebreak a little more consistent?

Reply:
I actually did not understand here...